biz.aQute.bnd.reporter/src/biz/aQute/bnd/reporter/plugins/entries/bundle/GogoPlugin.java

```diff
@@ -283,9 +285,8 @@ public Object extract(final Jar jar, final Locale locale) throws Exception {
 				gogoMethodDTO.title = m.getName();
 				gogoMethodDTO.parameters = Lists.newArrayList();
 
-				Optional<Annotation> oDescriptorAnnotation = m.annotations("*")
-					.filter(a -> ANNOTOATION_NAME_DESCRIPTOR.equals(a.getName()
-						.toString()))
+				Optional<Annotation> oDescriptorAnnotation = m
+					.annotations(ANNOTOATION_NAME_DESCRIPTOR.replace(".", "/"))
```

Comment:
`.annotations(Descriptors.fqnToBinary(ANNOTATION_NAME_DESCRIPTOR))`

Reply:
Thank you! Done.